src/libraries/System.Formats.Tar/tests/TarTestsBase.cs

```diff
@@ -115,6 +117,11 @@ public enum TestTarFormat
 
         protected static bool IsNotLinuxBionic => !PlatformDetection.IsLinuxBionic;
 
+        protected TarTestsBase()
+        {
+            CreateDirectoryDefaultMode = PlatformDetection.IsWindows ? UnixFileMode.None : Directory.CreateDirectory(GetRandomDirPath()).UnixFileMode;
```

Comment:
Is the default directory creation mode different between Unix flavors? I ask because I wonder if the value could be hardcoded to a const, like we did with `DefaultFileMode`, `DefaultDirectoryMode`, etc.

Reply:
> Is the default directory creation mode different between Unix flavors?

The default creation mode depends on the umask, it is: `0777 & ~umask`.